### ISSUE TITLE
Different encode step syntax than dragonfly gem

### ISSUE THREAD

**Issue by felixbuenemann** [cloud8421/dragonfly-server#13]
The current version of the dragonfly gem serialized the encode processor as:

``` ruby
 ["p", "encode", "jpg", "-quality 50"]
```

But dragonfly-server uses:

``` ruby
 ["e", "jpg", "-quality 50"]
```

Is there a reason for the different syntax or is it a bug in the elixir implementation?

I already have a fix for this locally, which currently supports both variants, but I'd like to know wether I should keep or drop support for the "e" syntax.

**Comment by cloud8421** [comment]
The original deployment of this app supports urls generate as far as 3 years ago on an older version, so support for the original format is needed. I'd say let's keep both variants.

**Comment by cloud8421** [comment]
Closed by #14
